voice/bxml/verbs/speakSentence.md

```diff
@@ -19,6 +19,13 @@ To choose a specific voice by name, use the `voice` attribute.
 | gender    | Selects the gender of the speaker. Valid values are `"male"` or `"female"`.<br><br>Default `"female"`                                                                            |
 | locale    | Selects the locale of the speaker. Consult `locale` column in the below table for valid values.<br><br>Default `"en_US"`                                                         |
 
+
+### Text Content
+| Name     | Description                                                                                                                                  |
+|:---------|:---------------------------------------------------------------------------------------------------------------------------------------------|
+| text | The text to speak. Cannot be blank. Can be a mixture of plain text and SSML tags (see below for list of supported tags). |
```

Comment:
So this is formatted as a table? Seems kind of awkward. Not sure how else to express it though...

Reply:
It is. You don't have to align the `|`

http://bw-dev-tdp-4169-empty-speak-sentence.s3-website-us-east-1.amazonaws.com/voice/bxml/verbs/speakSentence.html is what it looks like on the page

Reply:
Actually I think I misread your question. It's 2 separate tables if that's what you were asking

Reply:
Yeah it looks good rendered. Just seems odd to me to make it a table when there's only one element in it

Reply:
I did it to mimick the play audio page https://dev.bandwidth.com/voice/bxml/verbs/playAudio.html

Reply:
I only like this if the voice table gets its own header. Right now it looks like the voice table is part of the "Text Content" section.

Reply:
updated